Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -124,12 +130,28 @@ open class Player: BaseObject {
     }
     
     @objc private func willEnterForeground() {
-        if isChromeless {
+        if chromelessMode {
             Logger.logDebug("forced play after return from background", scope: "Player")
             play()
         }
     }
     
+    private func updateChromelessModeVisibility() {
```

Comment:
As the real Chromeless state is stored on `Core`, this code should probably be there. It's called as a result of changing the `Core.chromelessMode` property. IMHO this is clear as the methods which are actually doing something are `Core` methods.